stores/planship.ts

```diff
@@ -85,15 +84,15 @@ export const usePlanshipStore = defineStore('planship', () => {
     const { ssrContext } = useNuxtApp();
     const apiClient = ssrContext ? createServerApiClient() : createBrowserApiClient();
 
-    const defaultSubscription = computed(() => subscriptions.value.length > 0 ? subscriptions.value[0] : undefined)
+    const defaultSubscription = computed(() => subscriptions.value?.[0])
 
     const entitlements = computed(() => new Entitlements(entitlementsDict.value))
 
     const currentPlanSlug = computed(() => {
-      return defaultSubscription.value ? defaultSubscription.value.plan.slug : ''
+      return defaultSubscription.value?.plan.slug
```

Comment:
Prevously, this was returning an empty string if there was no slug. Now it'll return undefined. If you want it to return a string again you can use `defaultSubscription.value?.plan.slug || ''`.

Reply:
I think that undefined is OK here, doesn't change the behavior